Data_Preprocessing/nbconverted/file_merge.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+# In[2]:
+
+
+# define input paths
```

Comment:
Consider adding a comment explaining the difference between `_sc_` and `_sc_fs_` files here and why they need to be merged.